natashasthilaire/Swampy-Sells
# Homepage and Search

### About
This PR adds items to the homepage for the demo, allows filtering by category from homepage, and adds search bar functionality

### Test Instructions
1. Go to `home` branch
2. Check that clicking the subcategories redirects you properly
3. Ensure that search returns the proper results

### Screenshots
![image](https://github.com/natashasthilaire/Swampy-Sells/assets/93110503/4b4952b4-e510-4255-a6ee-f4bdf0a7ddd8)
![image](https://github.com/natashasthilaire/Swampy-Sells/assets/93110503/62827caf-8be9-4725-9e8d-43d943d01477)


### Closes Tickets
[SS-43](https://swampysells.atlassian.net/browse/SS-43)
